src/frontend/http.rs

```diff
@@ -2,25 +2,44 @@ use std::{net::SocketAddr, str::FromStr};
 
 use anyhow::Result;
 use axum::{
-    body::Bytes,
-    extract::Path,
+    extract::{BodyStream, Path},
     http::StatusCode,
     response::{IntoResponse, Response},
     routing::{delete, get, post},
     Router,
 };
+use futures_util::{
+    stream::{self},
+    StreamExt,
+};
 use log::info;
-
 use tower_http::cors::CorsLayer;
 
 use crate::{
-    backend::fs::{delete_file, read_file, write_file},
+    backend::fs::{delete_file, read_file, write_file, FileStream},
```

Comment:
I need to clarify Type Alias as a todo!
It have seen it in theory only.
since "A type alias defines a new name for an existing [type](https://doc.rust-lang.org/reference/types.html)"
The question I have the is, why do we need it here? 
Just for context?

Reply:
It helps us explicitly define the type, which makes the code more type-safe.